kiota_serialization_json/_version.py

```diff
@@ -1 +1 @@
-VERSION: str = '1.3.0'
+VERSION: str = '1.4.0'
```

Comment:
nit: a patch bump would have been sufficient according to the semver conventions.